documentation/content/101 Quick Start/getting-new-releases.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: Getting Notified About New Releases
+Tags: pelican-theme, new-releases
+Category: 101 — Quick Start
+Date: 2019-12-05
+Slug: getting-notified-about-new-releases
+Comment_id: qryv67o-getting-notified-about-new-releases
+authors: Jack De Winter
+---
+
+Unlike a number of themes that exist for Pelican, the Elegant theme is currently under
+active development. The immediate benefit of this for you is that Elegant supports
+recent technologies/services such as [Utterances](https://utteranc.es/) for comments and
+the [Applause button](https://applause-button.com/) for quick and simple user feedback.
+The other main benefits are that any requests, be they bug reports or feature requests,
+are logged in the project's Issues database.
+
+These issues are actively going through triage, being worked on by the Elegant community,
+and being addressed in ongoing releases of Elegant. In addition,
+[the roadmap](https://github.com/Pelican-Elegant/elegant/milestones) let's you see what
+the Elegant team is working on, allowing you to decide if you want to contribute and
+help bring some feature of the theme to life!
+
+To give you the control to decide whether to update your website and benefit from the
+latest version of Elegant, we offer two automated ways to find out about new releases.
+
+## Subscribe to the email Newsletter
+
+Each of the article pages on this Elegant documentation website contain a section at
+the bottom of the right sidebar titled `Get New Release Alert`. By adding your email
+address in the box below and pressing the `Notify Me` button, you are signing up for our
+newsletter. This simple newsletter uses Elegant's support for
+[MailChimp](https://mailchimp.com/), <!-- yaspeller ignore -->
+and is only used to notify people when there are new version of Elegant available.
+
+## Subscribe to the RSS feed for the Documentation Site
+
+If you have any reservations about actively subscribing to a newsletter through a
+mailing list, you can passively subscribe to receive updates notices for Elegant by
+subscribing to the RSS feed for this website using your browser. By selecting the RSS
+Feed icon under `Stay In Touch`, your browser will handle the information according to
+any plugins present in your browser.
+
+Most browsers will require you to install a plugin or extension to enable you to read and
```

Comment:
Instead we should tell them to copy the link and add it to their RSS readers. 

No addon or extension is required at all.

Reply:
fixed.